packages/turf-projection/test/out/mercator-fiji.geojson

```diff
@@ -6,7 +6,7 @@
 		"coordinates": [
 			[
 				[
-					-20031393.340521,
+					20043623.345058,
```

Comment:
to generate the expected coordinate I used ogr2ogr: 
`ogr2ogr  -f "GeoJSON" /vsistdout/ -t_srs epsg:900913 -s_srs epsg:4326 test/wgs84/polygon.geojson | jq '.features'` for most cases exept `geometry.geojson` where the jq exepression would be `jq '.features[0].geometry'`

I didn't find any online tool to visualize the coordinates so i copy pasted the geojson example from openlayer and put it in a jsfiddle here:

https://jsfiddle.net/z1e5bqLx/1/

Reply:
not familliar with the package, but it seems like a wrapper around gdal's ogr2ogr, it's easy to use it like `proj4` was used.
However keep in mind that `gdal` is not a funny dependency to install on different `OS` and adding this dependency to the test would require gdal to be install, maybe i could create another command like : 

`npm generate-toMercator-tests`

`npm generate-toWgs84-tests`

I'm not sure if org2org npm package checks for `gdal` at installation time or at runtime, but if it checks it at runtime that would make `gdal` required only for those command and not at installation time.

I just want you to be aware about the implications of depending on `gdal`

Reply:
So do you want to depend on `gdal` ? 
Should i create additional tests generation npm `scripts` ?

Reply:
I don't think it's a good idea, it might become a blocker for possible contributors...
Why not just restoring the `proj4` testing code and simply add a separate test suite for the crossing 180 meridian case using the files you generated with `org2org`? With a proper comment/documentation of why that is necessary.

@morganherlocker @rowanwins what do you think?